tests/test_client.py

```diff
@@ -404,16 +404,21 @@ async def test_e2e_example():
     ),
 )
 async def test_e2e_consume_multiple_topics(monitoring_enabled):
-    from examples.consume_multiple_topics import produce, stream_engine, topics
+    consume_multiple_topics_example = importlib.import_module(
```

Comment:
oof

Reply:
Indeed, but necessary if we want to keep the naming of the examples intact 😅